### PR TITLE
Updating to exclude authenticating via email/token because it doesn't work

### PR DESCRIPTION
**Pull request by saitwilio**
### Proposed changes
Updated Zendesk docs to exclude authenticating via email/token because several customers have reported it doesn't work.

### Merge timing
Once approved

### CODE REVIEW

File: src/connections/destinations/catalog/zendesk/index.md [hunk 1]
```diff
@@ -13,9 +13,7 @@ id: 54521fdc25e721e32a72ef06
 
 1. From the Segment web app, click **Catalog**.
 2. Search for "Zendesk" in the Catalog, select it, and choose which of your sources to connect the destination to.
-3. There are two ways to authenticate your Zendesk account with Segment:
-   * Use the standard email and password you use to Sign In to your Zendesk account. In the Zendesk settings, add your email in the **Email** setting and your password in the **Password** setting.
-   * Use Zendesk OAuth with a unique token. Get the corresponding token from your Zendesk account: **Settings > Channels > API** and under the Settings Tab choose the corresponding token from the "Active API Tokens" list. In the Zendesk settings, add your `email/token` in the **Email** setting (for example, `peter@intech.com/token` - use the actual word token in your email address) and add the actual token in the **Password** setting.
+3. Enter your Zendesk domain and click the 'Connect' button - you'll be redirected to Zendesk OAuth. Use the standard email and password you use to Sign In to your Zendesk account. In the Zendesk settings, add your email in the **Email** setting and your password in the **Password** setting.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/zendesk/index.md:16]:
```suggestion
3. Enter your Zendesk domain and click **Connect**. The Zendesk OAuth login opens in a new tab. Sign in with your Zendesk credentials to authenticate. In the Zendesk settings, add your email in the **Email** setting and your password in the **Password** setting.
```
@saitwilio 
Reworded slightly for clarity.

Is the sentence that begins with "In the Zendesk settings..." still applicable if using OAuth? I don't see fields for these in the destination settings in Segment.

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/zendesk/index.md (hunk 1 above)
Comment (saitwilio) [reply src/connections/destinations/catalog/zendesk/index.md:16]: You're right.. that sentence needs to be taken out. Good catch!

File: src/connections/destinations/catalog/zendesk/index.md [hunk 2]
```diff
@@ -13,9 +13,7 @@ id: 54521fdc25e721e32a72ef06
 
 1. From the Segment web app, click **Catalog**.
 2. Search for "Zendesk" in the Catalog, select it, and choose which of your sources to connect the destination to.
-3. There are two ways to authenticate your Zendesk account with Segment:
-   * Use the standard email and password you use to Sign In to your Zendesk account. In the Zendesk settings, add your email in the **Email** setting and your password in the **Password** setting.
-   * Use Zendesk OAuth with a unique token. Get the corresponding token from your Zendesk account: **Settings > Channels > API** and under the Settings Tab choose the corresponding token from the "Active API Tokens" list. In the Zendesk settings, add your `email/token` in the **Email** setting (for example, `peter@intech.com/token` - use the actual word token in your email address) and add the actual token in the **Password** setting.
+3. Enter your Zendesk domain and click **Connect**. The Zendesk OAuth login opens in a new tab. Sign in with your Zendesk credentials to authenticate. In the Zendesk settings, add your email in the **Email** setting and your password in the **Password** setting.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/zendesk/index.md:16]: > In the Zendesk settings, add your email in the **Email** setting and your password in the **Password** setting.

@saitwilio Can you clarify a bit on where these fields are supposed to be? I'm not following the step as described here. Once authentication with OAuth is done, is there still a need to add Email and Password?

Comment (saitwilio) [reply src/connections/destinations/catalog/zendesk/index.md:16]: Hi @markzegarelli I'm not sure what the steps look like exactly - I'm changing the docs requested by someone else, let me get on a chat with them to confirm.

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**